### DISCUSSION TITLE
winget keeps redownloading/installing the same version

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/winget-cli#2248] [Q&A]
I've noticed that when I `winget upgrade --all`, a couple items always show up with the same version and winget downloads them and reinstalls them. Is this the correct behavior?

Ultimaker Cura [Ultimaker.Cura] Version 5.0.0
CrystalDiskInfo (I can't remember the version)

**Comment by Masamune3210** [comment]
No its not, there is a chance either something is going wrong with the upgrade or the installers are not updating the version numbers for some reason

**Reply by myokeeh** [reply]
I assume it's not something I can fix then? How do you even go about reporting this to the devs? It seems like a pain.

**Reply by Masamune3210** [reply]
By doing exactly what you have basically, but instead of writing in a discussion, make a issue and fill out all information that it asks for and anything else you think could be useful
